config/application.rb

```diff
@@ -12,7 +12,8 @@ module BlogApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
-
+    config.autoload_lib(ignore: %w(assets tasks))
+    config.autoload_paths += %W({config.root}/lib/modules)
     # Configuration for the application, engines, and railties goes here.
     #
     # These settings can be overridden in specific environments using the files
```

Comment:
- You have done great so far :+1: :100: I was able to test your project and it works as expected, But there are a few issues that you need to fix before moving to the next milestone. :+1: 

- [ ] When I run `rails db:create` I get this long error as shown in the screenshot below, Upon looking into it, It appears to be related to the autoload_lib method, which doesn't exist in the context of your Rails application. The custom configuration you have in `application.rb` is what causing the issue. I Commented them out and I was able to run the application normally. I'll suggest you comment out `lines 15 and 16` as your app crashes and can't start while they are there and your app runs fine when they are removed. :smile: :wave: :+1:  :small_airplane: 

- - Screenshot reference:
![Selection_025](https://github.com/MenendezON/Blog-app/assets/107632780/884298dd-7bb4-4843-9f13-5d089ee6c494)